src/Init/Data/Nat/Bitwise/Lemmas.lean

```diff
@@ -324,6 +324,28 @@ theorem testBit_one_eq_true_iff_self_eq_zero {i : Nat} :
     Nat.testBit 1 i = true ↔ i = 0 := by
   cases i <;> simp
 
+theorem testBit_two_pow {n m : Nat} : Nat.testBit (2 ^ n) m = if n = m then true else false := by
```

Comment:
```suggestion
theorem testBit_two_pow {n m : Nat} : Nat.testBit (2 ^ n) m = if n = m then true else false := by
```
The `simp` normal form is `⊢ (2 ^ n).testBit m = decide (n = m)`, and I think this should perhaps even live in `Nat/Bitwise/Lemmas.lean`, but I leave that up to Kim :)

Reply:
Resolved.